google-api-client/pom.xml

```diff
@@ -36,20 +36,42 @@
         <configuration>
           <archive>
             <manifest>
-              <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+<addDefaultImplementationEntries>true</addDefaultImplementationEntries>
             </manifest>
+             <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
           </archive>
         </configuration>
         <executions>
           <execution>
             <id>jar</id>
-            <phase>compile</phase>
+            <phase>package</phase>
```

Comment:
Just remove this executions section like was done here:

https://github.com/google/google-oauth-java-client/commit/57fd88b34bea69005c0a3bf0ad547a7a3acb2005

Also, like in the above commit, remove the `phase` from maven-source-plugin below.